R/db-odbc.R

```diff
@@ -0,0 +1,24 @@
+#' @export
+db_desc.OdbcConnection <- function(x) {
+  info <- DBI::dbGetInfo(x)
+
+  host <- if (info$servername == "") "localhost" else info$servername
+  port <- if (info$port == "") "" else paste0(":", port)
+
+  paste0(
+    info$dbms.name, " ", info$db.version,
+    "[", info$username, "@", host, port,
+    "/", info$dbname , "]")
+
+}
+
+#' @export
+sql_escape_ident.OdbcConnection <- function(con, x) {
+  x
```

Comment:
It's really important to properly escape the identifiers. It does make the SQL longer than what you'd usually write by hand, but it protects the user against knowing all the special SQL keywords and ensure it works with unnamed `summarise()` which creates variable names like `mean(x)`